.github/workflows/deploy.yml

```diff
@@ -130,10 +130,10 @@ jobs:
           push: true
 
       - name: Push distroless image
-        uses: docker/build-push-action@v2
+        uses: docker/build-push-action@v3
         with:
           context: distroless/${{ matrix.platform }}/target/docker/stage
           file: distroless/${{ matrix.platform }}/target/docker/stage/Dockerfile
-          platforms: linux/amd64,linux/arm64/v8
```

Comment:
So we drop arm support ?

Reply:
Whoops, I deleted that by accident (just while testing).  I'll put it back again.